Demo/Color+Helpers.swift

```diff
@@ -0,0 +1,41 @@
+import UIKit
+
+public extension UIColor {
```

Comment:
You have to also copy the frameworks on build using a `copy files` step. 👍 

Also remember to put the dependency inside `Cartifle.private` 😄